airbyte-integrations/connectors/source-surveymonkey/integration_tests/abnormal_state.json

```diff
@@ -9,22 +9,243 @@
+      "stream_state": {
+        "states": [
+          {
+            "partition": {
+              "survey_id": "306079584"
+            },
+            "cursor": {
+              "date_modified": "2025-02-27 12:39:31"
+            }
+          },
+          {
+            "partition": {
+              "survey_id": "307785429"
+            },
+            "cursor": {
+              "date_modified": "2025-02-27 12:39:32"
+            }
+          },
+          {
+            "partition": {
+              "survey_id": "307785444"
+            },
+            "cursor": {
+              "date_modified": "2025-02-27 12:39:33"
+            }
+          },
+          {
+            "partition": {
+              "survey_id": "307785394"
+            },
+            "cursor": {
+              "date_modified": "2025-02-27 12:39:34"
+            }
+          },
+          {
+            "partition": {
+              "survey_id": "307785402"
+            },
+            "cursor": {
+              "date_modified": "2025-02-27 12:39:36"
+            }
+          },
+          {
+            "partition": {
+              "survey_id": "307785408"
+            },
+            "cursor": {
+              "date_modified": "2025-02-27 12:39:37"
+            }
+          },
+          {
+            "partition": {
+              "survey_id": "307784834"
+            },
+            "cursor": {
+              "date_modified": "2025-02-27 12:39:38"
+            }
+          },
+          {
+            "partition": {
+              "survey_id": "307785448"
+            },
+            "cursor": {
+              "date_modified": "2025-02-27 12:39:40"
+            }
+          },
+          {
+            "partition": {
+              "survey_id": "307784863"
+            },
+            "cursor": {
+              "date_modified": "2025-02-27 12:39:41"
+            }
+          },
+          {
+            "partition": {
+              "survey_id": "307784846"
+            },
+            "cursor": {
+              "date_modified": "2025-02-27 12:39:42"
+            }
+          },
+          {
+            "partition": {
+              "survey_id": "307784856"
+            },
+            "cursor": {
+              "date_modified": "2025-02-27 12:39:43"
+            }
+          },
+          {
+            "partition": {
+              "survey_id": "307785388"
+            },
+            "cursor": {
+              "date_modified": "2025-02-27 12:39:45"
+            }
+          },
+          {
+            "partition": {
+              "survey_id": "307785415"
+            },
+            "cursor": {
+              "date_modified": "2025-02-27 12:39:46"
+            }
+          },
+          {
+            "partition": {
+              "survey_id": "510388524"
+            },
+            "cursor": {
+              "date_modified": "2025-02-27 12:39:48"
+            }
+          }
+        ]
+      }
+    },
+    "data": {
```

Comment:
It's just how state message return and it's not right to have partial message in it.

Reply:
I'm surprised because when I check a connection state in production, I don't see the `data` part. For example, here is a connection on my test workspace: https://cloud.airbyte.com/workspaces/8ba9a84d-ef49-41fb-9aaa-9f75fb0f351e/connections/c72782d2-6a32-4408-8c43-421af5d2fca4/settings How did you get this state message? Is this what the platform sends as an input to the source?